src/pages/About/AboutKyberSwap.tsx

```diff
@@ -893,7 +870,7 @@ function AboutKyberSwap() {
           >
             <BtnPrimary
               as={Link}
-              to={createPoolLink}
+              to={'/pools?tab=elastic'}
```

Comment:
This is wrong, you changed from create pool `'/create/...'` to pools list `'/pools...'`

Reply:
If there is any product requirement changed, just let us know. I'm reviewing on comparing with original production's code

Reply:
Sure man